src/server/detail/wrapped_json_path.h

```diff
@@ -59,19 +59,34 @@ template <typename T> void OptionalEmplace(bool keep_defined, T src, T* dest) {
 
 }  // namespace details
 
+enum class JsonPathType { V2, Legacy /*Or V1*/ };
```

Comment:
```suggestion
enum class JsonPathType { kV2, kLegacy /*Or kV1*/ };
```
This is our convention for constants (see Google Style Guide)